library/ssl_tls13_client.c

```diff
@@ -1440,11 +1440,124 @@ static int ssl_tls13_read_server_hello( mbedtls_ssl_context *ssl )
 }
 
 /*
- * Handler for MBEDTLS_SSL_ENCRYPTED_EXTENSIONS
+ *
+ * EncryptedExtensions message
+ *
+ * The EncryptedExtensions message contains any extensions which
+ * should be protected, i.e., any which are not needed to establish
+ * the cryptographic context.
+ */
+
+/*
+ * Overview
  */
-static int ssl_tls13_read_encrypted_extensions( mbedtls_ssl_context *ssl )
+
+/* Main entry point; orchestrates the other functions */
+static int ssl_tls13_encrypted_extensions_process( mbedtls_ssl_context *ssl );
+
+static int ssl_tls13_encrypted_extensions_parse( mbedtls_ssl_context *ssl,
+                                           const unsigned char *buf,
+                                           size_t buf_len );
+static int ssl_tls13_encrypted_extensions_postprocess( mbedtls_ssl_context *ssl );
+
+/*
+ * Handler for  MBEDTLS_SSL_ENCRYPTED_EXTENSIONS
+ */
+static int ssl_tls13_encrypted_extensions_process( mbedtls_ssl_context *ssl )
```

Comment:
rename to `ssl_tls1_3_read_encrypted_extensions`

Reply:
why? seems "read" can't represent the reall meaning of what we do in this function.

Reply:
Rebase to latest dummy handler PR, and rename the xxx_read_xxx() to xxx_process_xxx().